### ISSUE TITLE
RoBERTa: A Robustly Optimized BERT Pretraining Approach

### ISSUE THREAD

**Issue by hs-sugi** [One-Day-One-Paper/One-Day-One-Paper#166]
- Yinhan Liu, Myle Ott, Naman Goyal, Jingfei Du, Mandar Joshi, Danqi Chen, Omer Levy, Mike Lewis, Luke Zettlemoyer, Veselin Stoyanov
- [Submitted on 26 Jul 2019]

### Abstract
- **Robustly Optimized BERT Pretraining approach**
- BERT가 data, hyperparameter 부분에서 undertrained된 것을 발견
- 기존의 BERT 모델을 유지하면서, 학습 단계의 hyper paramter들을 조정하여 성능을 높이는 방법
- RoBERTa Summary 
  - 160GB의 데이터 
  - Dynamic masking 
  - MLM만으로 pre-train 
  - Full-sentence 형식의 input 
  - BERT의 약 32배의 batch size 
  - byte-level BPE tokenizer

![image](https://user-images.githubusercontent.com/40443049/167288301-8030e9db-9e44-4cf4-a387-50d421edbbca.png)
![image](https://user-images.githubusercontent.com/40443049/167288307-ab781dd9-e7a5-4126-805b-56c0493dd5d3.png)
![image](https://user-images.githubusercontent.com/40443049/167288313-5ecf7927-a76b-4eea-a399-69b81b0abd54.png)
![image](https://user-images.githubusercontent.com/40443049/167288316-22bd9421-4137-48f0-95af-33c2c44933fe.png)
![image](https://user-images.githubusercontent.com/40443049/167288323-ac92becf-9089-4496-b5e0-07ed7ed32c97.png)
